DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Interfaces/IRhinoPlugin.cs

```diff
@@ -0,0 +1,7 @@
+namespace Speckle.Connectors.Rhino7.Interfaces;
```

Comment:
If they start to look like this, we can probably make this an ISpecklePlugin...  let's watch the space...